rules/detekt/src/test/kotlin/io/nlopez/compose/rules/detekt/LambdaParameterInRestartableEffectCheckTest.kt

```diff
@@ -78,4 +78,65 @@ class LambdaParameterInRestartableEffectCheckTest {
         val errors = rule.lint(code)
         assertThat(errors).isEmpty()
     }
+
+    @Test
+    fun `error out when detecting a lambda named onDispose used in a non-DisposableEffect`() {
```

Comment:
you need to add these tests to ktlint's `LambdaParameterInRestartableEffectCheckTest.kt` as well.

Reply:
Good catch, done!